cylc/flow/parsec/example/test.py

```diff
@@ -29,7 +29,7 @@
 sys.path.append(
     os.path.join(os.path.dirname(os.path.abspath(__file__)), '../..'))
 
-cylc.flow.flags.verbose = True
+cylc.flow.flags.verbosity = 1
 
 
 class Testcfg(ParsecConfig):
```

Comment:
What is this file actually for?

Reply:
No idea whatsoever, and that is definitely not the right way to do it (will change the value for all tests).

Reply:
@hjoliver Can we get rid of this file?

Reply:
Lost in the mists of time, probably some built-in test during original parsec development Yes, delete it!

Reply:
(I'll push a file deletion patch to this branch, and then merge...)